kyco/core/buffers.py

```diff
@@ -6,7 +6,7 @@
 
 __all__ = ['KycoBuffers']
 
-log = logging.getLogger('Kyco')
+log = logging.getLogger(__name__)
```

Comment:
Using `__name__` is considered a good convention by [Python docs](https://docs.python.org/3/howto/logging.html#logging-advanced-tutorial).